prow/github/types.go

```diff
@@ -464,6 +464,100 @@ type Commit struct {
 	Modified []string `json:"modified"`
 }
 
+// SingleCommit is the commit part received when requesting a single commit
+// https://developer.github.com/v3/repos/commits/#get-a-single-commit
+type SingleCommit struct {
```

Comment:
Similarly, can we keep only the fields we care about?

Reply:
will do